heft-plugins/heft-typescript-plugin/src/TypeScriptBuilder.ts

```diff
@@ -1090,7 +1119,11 @@ export class TypeScriptBuilder {
     if (tsconfig.options.incremental) {
       compilerHost = ts.createIncrementalCompilerHost(tsconfig.options, system);
     } else {
-      compilerHost = ts.createCompilerHost(tsconfig.options, undefined, system);
+      compilerHost = (ts.createCompilerHostWorker ?? ts.createCompilerHost)(
```

Comment:
Curious, what is this for? Is this a fix that we just never had implemented?

Reply:
The current installed version of TypeScript, `system` is not an argument to `createCompilerHost` but is an accepted argument to the underlying `createCompilerHostWorker` call.